src/hooks/useInitializeWallet.ts

```diff
@@ -61,8 +59,8 @@ export default function useInitializeWallet() {
       try {
         PerformanceTracking.startMeasuring(PerformanceMetrics.useInitializeWallet);
         logger.debug('[useInitializeWallet]: Start wallet setup');
-        await resetAccountState();
-        logger.debug('[useInitializeWallet]: resetAccountState ran ok');
+        dispatch(requestsResetState());
```

Comment:
the `resetAccountState` function only has the WC v1 requests resetting logic left - so using this function directly instead